app/charts/shared/chart-helpers.spec.tsx

```diff
@@ -140,28 +137,6 @@ describe("useQueryFilters", () => {
   });
 });
 
-describe("getWideData", () => {
-  const exampleMap: InternMap<string, Observation[]> = new Map();
-  exampleMap.set("2021-01-02", [{ segment: "abc", value: 1 }]);
-  exampleMap.set("2015-03-03", [{ segment: "abc", value: 10 }]);
-  exampleMap.set("2028-12-12", [{ segment: "abc", value: 12 }]);
-
-  it("should return sorted data", () => {
-    const wideData = getWideData({
-      dataGroupedByX: exampleMap,
-      xKey: "date",
-      getY: (d: Observation) => Number(d["value"]),
-      getSegment: (d: Observation) => String(d["segment"]),
-    });
-
-    expect(wideData.map((d) => d["date"])).toEqual([
-      "2015-03-03",
-      "2021-01-02",
-      "2028-12-12",
-    ]);
-  });
-});
```

Comment:
Why is the test removed here ?

Reply:
I think it was done because now we expect the data to come in sorted, so this test would not pass in its current form. I will try to add a new test to ensure that this is the case 👍